optimizely/config_manager.py

```diff
@@ -249,6 +258,31 @@ def set_update_interval(self, update_interval):
 
         self.update_interval = update_interval
 
+    def set_blocking_timeout(self, blocking_timeout):
+        """ Helper method to set time in seconds to block the config call until config has been initialized.
+
+        Args:
+          blocking_timeout: Time in seconds to block the config call.
+        """
+        if not blocking_timeout:
```

Comment:
nit. Check if it's None. Otherwise, this will be true when blocking_timeout is 0